util/nbaUtils.js

```diff
@@ -90,6 +90,23 @@ getGamesInWeekTest = function(data, lastPlayedIndex, daysInWeek) {
     
 }
 
+/**
+ * @param {Object[]} data JSON object of players in a team
+ * @return {Object[]} first and last name of all players

```

Comment:
should we mention that the return has json in it?

Reply:
Nah. It doesn't really have json; it parses the json for the name and puts it into an object, which is why i put Object[] as the return type.